Apps/APN-UIKit/APN UIKit/APN UIKit/View/Extension/ViewControllerExtension.swift

```diff
@@ -0,0 +1,15 @@
+import Foundation
+import UIKit
+
+extension UIViewController {
+    
+    func showInfoAlert(withMessage message: String) {
+        let dialogMessage = UIAlertController(title: "Info", message: message, preferredStyle: .alert)
```

Comment:
```suggestion
    func showInfoAlert(withMessage message: String, title: String = "Info") {
        let dialogMessage = UIAlertController(title: title, message: message, preferredStyle: .alert)
```

Great extension function! I liked that you added it. 

This is a suggestion to make it more flexible, if we ever need it.